e2e/api/steps_LRspecCui.js

```diff
@@ -159,13 +168,37 @@ module.exports = {
 
     await waitForFinishedBusinessProcess(caseId);
   },
+
+  requestJudgement: async (user, response = 'FULL_ADMISSION', scenario = 'ONE_V_ONE') => {
+
+    await apiRequest.setupTokens(user);
+
+    eventName = 'REQUEST_JUDGEMENT_ADMISSION_SPEC';
+    caseData = await apiRequest.startEvent(eventName, caseId);
+    let requestJudgementData = eventData['requestJudgement'][scenario][response];
+    // requestJudgementData = await replaceClaimantResponseWithCourtNumberIfCourtLocationDynamicListIsNotEnabled(requestJudgementData);
```

Comment:
Removed commented out code here and below

Reply:
Updated